packages/vue/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@glidejs/glide": "^3.3.0",
-    "@storefront-ui/shared": "^0.6.3",
```

Comment:
@patzick is changed by lerna, can you remove this change

Reply:
No, Lerna only bumps the number, doesn't interfere with scope. This is the reason why going back to the previous version doesn't work and prevents this kind of blockers for the future. So this line is a crucial part of this fix.

Reply:
right, thanks :)